From 695c4f3ba589980aa793c4472c61cf17fe909801 Mon Sep 17 00:00:00 2001
From: Junping Du <junping_du@apache.org>
Date: Thu, 28 Jan 2016 06:14:21 -0800
Subject: [PATCH 2362/2848] CDH-50004 MAPREDUCE-6595. Fix findbugs warnings in
 OutputCommitter and FileOutputCommitter.
 Contributed by Akira AJISAKA.

(cherry picked from commit 86560a4978c498bd5e892c509a466807a812b8a7)

Change-Id: I3ff534791eb7c07bd4130ff30e9e52d9cae6ec41
---
 .../dev-support/findbugs-exclude.xml               |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/hadoop-mapreduce-project/dev-support/findbugs-exclude.xml b/hadoop-mapreduce-project/dev-support/findbugs-exclude.xml
index dd4892b..43fe05b 100644
--- a/hadoop-mapreduce-project/dev-support/findbugs-exclude.xml
+++ b/hadoop-mapreduce-project/dev-support/findbugs-exclude.xml
@@ -159,6 +159,7 @@
        <Method name="commitJob" />
        <Method name="recoverTask" />
        <Method name="isRecoverySupported" />
+       <Method name="isCommitJobRepeatable" />
        </Or>
        <Bug pattern="NM_WRONG_PACKAGE" />
      </Match>
@@ -170,6 +171,7 @@
        <Method name="cleanupJob" />
        <Method name="recoverTask" />
        <Method name="isRecoverySupported" />
+       <Method name="isCommitJobRepeatable" />
        </Or>
        <Bug pattern="NM_WRONG_PACKAGE_INTENTIONAL" />
      </Match>
-- 
1.7.9.5

